contrib/solaar/template.py

```diff
@@ -1,22 +1,34 @@
 pkgname = "solaar"
-pkgver = "1.1.9"
+pkgver = "1.1.10"
 pkgrel = 0
-build_style = "python_module"
-hostmakedepends = ["python-setuptools"]
+build_style = "python_pep517"
+hostmakedepends = ["python-build", "python-wheel", "python-installer"]
 depends = [
     "python-evdev",
     "python-pyudev",
     "python-psutil",
     "python-pyyaml",
     "python-xlib",
     "python-gobject",
+    "python-dbus",
 ]
-checkdepends = list(depends)
+checkdepends = ["python-pytest"] + depends
```

Comment:
if it has no tests, what's the deal with this stuff

Reply:
well I saw
```
=> solaar-1.1.10-r0: running do_check...
=> solaar-1.1.10-r0: ERROR: pytest not found
```
but I'll drop it since `!check` alone avoids it anyway

Reply:
just remove the checkdepends line entirely

Reply:
also removing setuptools from hostdeps is wrong

Reply:
just noticed setuptools in hostdeps wasn't needed anymore to even "cross-build" which is why I tried that